examples/01-calculations.md

```diff
@@ -121,12 +123,15 @@ Finally we will actually run this specification.
 
 Even though the notebook is blocked, execution of `add` is actually happening _on the cluster_.
 
-If you inspect the (verbose) logs you'll see "copying file/folder" + Slurm-related stuff
-
 ```python
 r = aiida.engine.run(z.on(env))
 ```
 
+If the execution of the cell above is hanging for too long, you may want to drop to the command line and inspect the running processes, e.g. using via `verdi process list`.
+The (verbose) daemon logs should be showing "copying file/folder" + Slurm-related stuff.
+
+If you're having trouble with remote execution, feel free to continue through the rest of the tutorial on your local computer by setting `env=local_env`.
```

Comment:
Running on 'local_env' won't work for the map jobs in this notebook (which require the bundled slurm scheduler

Reply:
I see, will remove it again.
Eventually, it might be nice to have an example notebook that does not require access to a remote slurm cluster.

Reply:
We still need to remove that line here.